src/webui/src/static/function.ts

```diff
@@ -118,33 +117,36 @@ const killJob = (key: number, id: string, status: string, updateList?: Function)
     })
         .then(res => {
             if (res.status === 200) {
-                message.destroy();
-                message.success('Cancel the job successfully');
+                // message.destroy();
+                // message.success('Cancel the job successfully');
                 // render the table
                 if (updateList) {
                     updateList();  // FIXME
                 }
             } else {
-                message.error('fail to cancel the job');
+                alert('fail to cancel the job');
+                // message.error('fail to cancel the job');
```

Comment:
remove unused function?

Reply:
this function will use in [kill a trial],
I 'll update this function in next commit in PR.
Thanks for reviewing.